components/compact-table-accordion-group/package.json

```diff
@@ -24,10 +24,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```

Comment:
This one was actually Jonathan Sheldrake, based off the `table-accordion-group` which was Alasdair Mcleay